x-pack/plugins/infra/public/components/waffle/waffle_node_type_switcher.tsx

```diff
@@ -21,48 +20,36 @@ interface Props {
   changeMetric: (metric: InfraMetricInput) => void;
 }
 
+const nodeOptions = [
```

Comment:
Originally this was a case of me taking the wireframes pretty literally. However, I've just had a look at `<EuiButtonGroup />` and the `label` prop must be a string, it doesn't support nodes. So without a change to the EUI component I'm not sure it would be trivial to have the images render inline with the text?

Reply:
@makwarth what was your intention when creating the mockup? should we keep the node type icons?

Looks like the `EuiButtonGroupOption` typing and the `propTypes` of the `EuiButtonGroup` are indeed more restrictive than the code, which actually passes through the `iconType` to the individual button. How about we fix EUI and then add back the icons in a separate PR if @makwarth agrees?

Reply:
My intention was not to include the icons. We can add them later if necessary, but I don't think we need them.